tests/php/test-amp-helper-functions.php

```diff
@@ -2576,6 +2588,10 @@ public function data_amp_has_paired_endpoint() {
 	 * @param bool   $is_amp
 	 */
 	public function test_amp_has_paired_endpoint_go_to( $paired_url_structure, $suffix, $is_amp ) {
+		if ( is_multisite() ) {
+			switch_to_blog( self::factory()->blog->create() );
+		}
```

Comment:
Why is this needed?

Reply:
Just to ensure the working with subdirectory installation i.e. on `/path1/some-post/amp`. `/path2/some-post/amp`. Or we can write another test to ensure this?

Reply:
Ah, ok. How about just adding a comment to indicate this is the reason for the blog switch?

Reply:
I have added a inline comment.